### PR TITLE
Added changes to MATLAB, PT, and TF docs

### PR DESCRIPTION
**Pull request by matthewjreynolds**
I updated the MATLAB docs from CMS to reflect the newer version of MATLAB, changed multiple instances of "Eagle" to "Kestrel," and updated interactive and batch job commands. 

For PyTorch and TensorFlow documentation I updated the module name for anaconda.

### CODE REVIEW

File: docs/Documentation/Applications/Matlab/batch.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ Once the job has finished, the standard output is saved in a file called
 #SBATCH --nodes=1                # Number of nodes
 #SBATCH --job-name=matlabTest              # Name of job
 #SBATCH --account=<account_string>        # Program-based WCID (account string associated with job)
+#SBATCH --partition=standard
```
Comment (yandthj) [inline docs/Documentation/Applications/Matlab/batch.md:56]: We're still recommending that users don't specify a partition in their job scripts unless using debug. Is there a specific reason for adding the standard partition?

File: docs/Documentation/Applications/Matlab/interactive.md [hunk 2]
```diff
@@ -10,11 +10,10 @@ Connection](https://www.nrel.gov/hpc/system-connection.html).
 
 ## Running MATLAB via an Interactive Job
 
-After connecting to the login node, the next step is to start an interactive job. For
-example, the following command gets # nodes for interactive use for an hour:
+After connecting to the login node, the next step is to start an interactive job. For example, the following command gets a user-selected number of nodes for interactive use, taking as input tasks per node, job duration, account, and partition.
 
 ```
-$ srun -N # --time 01:00:00 --account=<handle> --pty $SHELL
+$ salloc --nodes=<number of nodes> --ntasks-per-node=<tasks per node> --account=<your account here> --time=<desired time> --partition=<desired partition>
```
Comment (yandthj) [inline docs/Documentation/Applications/Matlab/interactive.md:16]: We're still recommending that users don't specify a partition in their job scripts unless using debug. I think the partition flag could be removed.

**Review by yandthj — CHANGES_REQUESTED** [review]
Thanks for the updates! I just have a couple notes. I can make the suggested changes for you if you confirm they are correct: 

- Should the example job script on the Machine_Learning/index.md file also have "module load conda" changed to "module load anaconda3"? 
- Can I remove the --partition flags from the job submission commands? We still generally recommend not specifying the partition

File: docs/Documentation/Applications/Matlab/batch.md (hunk 1 above)
Comment (matthewjreynolds) [reply docs/Documentation/Applications/Matlab/batch.md:56]: I was getting weird behavior during my tests when I did not specify partitions, so I was adding them to be safe. Feel free to remove them.

File: docs/Documentation/Applications/Matlab/interactive.md (hunk 2 above)
Comment (matthewjreynolds) [reply docs/Documentation/Applications/Matlab/interactive.md:16]: Sae as above: the partition flag can be removed.

File: docs/Documentation/Applications/Matlab/batch.md (hunk 1 above)
Comment (matthewjreynolds) [reply docs/Documentation/Applications/Matlab/batch.md:56]: I removed the specification of partition from the batch file and changed module load conda to module load anaconda3.

File: docs/Documentation/Applications/Matlab/interactive.md (hunk 2 above)
Comment (matthewjreynolds) [reply docs/Documentation/Applications/Matlab/interactive.md]: I removed the partition specification.

**Comment by matthewjreynolds** [comment]
Hi Haley,

I went through the MATLAB documentation and removed all the specifications of partition in the batch files and sallow commands. I also changed the machine learning index page so that: 1) the partition was not specified and 2) module load anaconda replaced module load conda.

**Review by yandthj — APPROVED** [review]

**Pull request merged**